testing/test_compat.py

```diff
@@ -26,6 +29,8 @@ def __repr__(self):
             return "<Evil left={left}>".format(left=self.left)
 
         def __getattr__(self, attr):
+            if attr == "__pytest_wrapped__":
```

Comment:
i just noticed this special case - we shouldn't have it - because in practice this means we blow up on evil objects there